react-forms/my-app/src/ControlledForm.tsx

```diff
@@ -0,0 +1,35 @@
+import { FormEvent, useState } from 'react';
+
+export function RegistrationFormControlled() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(event: FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    console.log('ControlledFormObject', { username, password });
+  }
+
+  return (
+    <form id="controlledForm" onSubmit={handleSubmit}>
```

Comment:
The `id` on all these tags are not needed. You don't query the elements in React, so id's are not all that useful in React.